data/scripts/actions/others/gooey_mass.lua

```diff
@@ -0,0 +1,38 @@
+local config = {
+	{chanceFrom = 0, chanceTo = 38}, -- nothing
+	{chanceFrom = 39, chanceTo = 2132, itemId = 2152, count = 2}, -- platinum coin
+	{chanceFrom = 2133, chanceTo = 4187, itemId = 15487, count = 10}, -- larvae
+	{chanceFrom = 4188, chanceTo = 6040, itemId = 2144, count = 2}, -- black pearl
```

Comment:
Gooey Mass seems to drop Orichalcum Pearl (x1) aswell, although there is no chance data for it .

https://tibia.fandom.com/wiki/Gooey_Mass

https://www.tibiawiki.com.br/wiki/Gooey_Mass

Reply:
yeah it might certainly be wrong, since it is not a "rare" item and it would appear on the statistics